src/map/options.ts

```diff
@@ -323,7 +323,7 @@ export class MapOptions extends OptionsGroup {
                 );
 
                 // set width first, since setting position can influence it
-                modalDialog.style.width = `${modalDialog.offsetWidth}px`;
+                modalDialog.style.width = '35%'; //`${modalDialog.offsetWidth / 1.2}px`;
```

Comment:
oh, I guess this is the origin of the small modal on small screens =). Why this change?

Reply:
Ok, I have reverted it back. The issue is that for narrow screen the modal is then outside of the screen. Also, I have noticed that when in iPad/iPhone mode, I cannot seem to move the modal.

Reply:
So, I have reverted the width change back to modalDialog.offsetWidth but slightly smaller. And then I added a check for the screen.width - so that when it is too narrow, the map setting will place itself rather in the middle and not on the right.